### ISSUE TITLE
Product #246 not found

### ISSUE THREAD

**Issue by ghost** [sparkfun/3D_Models#259]
Product #246 isn't found - I was just wandering and found the fact that the product had a 3d model.

**Comment by santaimpersonator** [comment]
I don't understand the issue you are trying to specify. Could you rephrase it?

**Comment by bboyho** [comment]
It looks like Elizabeth made changes to [13656](https://github.com/sparkfun/3D_Models/tree/master/products/13656) and pushed it to the main repo. That is a stepper motor.

I'm not sure if the customer was confused on the branch name or maybe they were looking for something else? I tried looking for a model for [product "246"](https://www.sparkfun.com/products/246) and that appears to be a PTH component (specifically the "Optical Detector / Phototransistor - QRD1114"). The product page had the 3D model icon was turned on but there wasn't one available in this repo. I turned it back off. I also accidentally typed 258 and which also had the 3D model icon turned on for the  "Basic 20x4 Character LCD - Black on Green 5V". However, this repo also did not have a model available for the part so I turned it off. 

Since the GitHub user account appears to deleted, I'm going to close this issue. ¯\_(ツ)_/¯
